Makefile

```diff
@@ -336,39 +336,43 @@ ci-coverage-report: $(SUMMARY_COVER_PROFILE) coverage-report
 	go install github.com/mattn/goveralls@v0.0.7
 	@goveralls -coverprofile=$(SUMMARY_COVER_PROFILE) -service=buildkite || (printf $(RED) "Generating report for Coveralls (goveralls) failed."; exit 1)
 
+# optional args to make it possible to create multiple keyspaces:
+# make install-schema TEMPORAL_DB=temporal2 VISIBILITY_DB=temporal_visibility2
+TEMPORAL_DB?=temporal
+VISIBILITY_DB?=temporal_visibility
 ##### Schema #####
 install-schema: temporal-cassandra-tool
 	@printf $(COLOR) "Install Cassandra schema..."
-	./temporal-cassandra-tool drop -k temporal -f
-	./temporal-cassandra-tool create -k temporal --rf 1
-	./temporal-cassandra-tool -k temporal setup-schema -v 0.0
-	./temporal-cassandra-tool -k temporal update-schema -d ./schema/cassandra/temporal/versioned
-	./temporal-cassandra-tool drop -k temporal_visibility -f
-	./temporal-cassandra-tool create -k temporal_visibility --rf 1
-	./temporal-cassandra-tool -k temporal_visibility setup-schema -v 0.0
-	./temporal-cassandra-tool -k temporal_visibility update-schema -d ./schema/cassandra/visibility/versioned
+	./temporal-cassandra-tool drop -k $(TEMPORAL_DB) -f
```

Comment:
They are `-default` and `-visibility` in `config.yaml`. Maybe change `TEMPORAL_DB` to `DEFAULT_DB`?